sdk-python/littlehorse/user_utils.py

```diff
@@ -100,3 +100,24 @@ def register_task(
             logging.info(f"TaskDef {name} already exits, skipping")
             return
         raise e
+
+
+def register_external_event(
```

Comment:
This is nice! Java and Go don't have it yet :wink:

Reply:
Actually, could you please rename it to `register_external_event_def`? The `def` is important, `ExternalEvent != ExternalEventDef`.

Reply:
is it possible "register" a event? isn't like post_external_event? 

register_external_event is for PutExternalEventDef
post_external_event is PutExternalEvent

or is it confuse? 🤔

Reply:
in lhctl we have postEvent

Reply:
This method creates external event definition, so it should have _def_ at the end.

You "postEvent` an External Event, which is dfferent than `externalEvent`.

It's `lhctl deploy exteranlEventDef` and `lhctl postEvent`... probably shouldbe `lhctl put externalEvent` but oops

Reply:
what is instead of register we use "create"